### PR TITLE
WIP: feat(iwd): introducing the iwd module

### PR DESCRIPTION
**Pull request by johannbg**
Introducing the iwd module

This pull request changes...

## Checklist
- [ ] I have tested it locally
- [ ] I have reviewed and updated any documentation if relevant
- [ ] I am providing new code and test(s) for it

Fixes #

### CODE REVIEW

**Comment by johannbg** [comment]
Needs a generator to configure the SSID and password for iwd from the kernel command line

**Comment by johannbg** [comment]
@dustymabe @bengal it would be nice to get some feedback how on the NM module will pick up and use/manage iwd in the initrd. This is not a problem with networkd since we are just using upstream type unit files however the NM module is somewhat special ( as opposed to just use the exact same type unit as is used in real root ) and I'm unsure if it's enough to just use the iwd_backend.conf file as you would do with NM 1.30+ and works fine on F34

File: modules.d/35iwd/module-setup.sh [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+installkernel() {
+    instmods '=drivers/net/wireless'
+}
+
+# Install the required file(s) and directories for the module in the initramfs.
+install() {
+
+    inst_rules "$moddir/99-iwd.rules"
+    inst_simple "$moddir/iwd.service" "$systemdsystemunitdir"/iwd.service
```
Comment (bengal) [inline modules.d/35iwd/module-setup.sh:42]: This file is not included in the commit.

**Comment by bengal** [comment]
> I'm unsure if it's enough to just use the iwd_backend.conf file as you would do with NM 1.30+ and works fine on F34

Yes, it should be enough to enable IWD in NM.

**Comment by polarathene** [comment]
I think you need to have the stale bot ignore PR / issues by using an explicit label to opt-out, then you won't have this open/close battle :sweat_smile:

**Pull request closed without merging**